6_visualize/src/combine_animation_frames.R

```diff
@@ -1,10 +1,10 @@
-combine_storm_frames <- function(gif_file, animation_cfg, task_names=NULL) {
+combine_animation_frames <- function(gif_file, animation_cfg, task_names=NULL) {
 
   # run imageMagick convert to build a gif
   if(is.null(task_names)) task_names <- '*'
-  png_files <- paste(sprintf('6_visualize/tmp/gif_frame_%s.png', task_names), collapse=',')
+  png_files <- paste(sprintf('6_visualize/tmp/gif_frame_%s.png', task_names), collapse=' ')
   magick_command <- sprintf('magick convert -delay %d -loop 0 %s %s',
-                            animation_cfg$frame_delay_cs, '6_visualize/tmp/gif_frame_*.png', gif_file)
+                            animation_cfg$frame_delay_cs, png_files, gif_file)
```

Comment:
:+1:
I don't see a better way of specifying *all* pngs that were generated that wouldn't rely on a dir and file name pattern like this. It feels a little different than how explicit we are elsewhere, but gotta do it.